src/mlpack/core/data/load.hpp

```diff
@@ -96,7 +96,11 @@ bool Load(const std::string& filename,
           arma::Mat<eT>& matrix,
           DatasetMapper<PolicyType>& info,
           const bool fatal = false,
-          const bool transpose = true);
+          const bool transpose = true)
+{
+  PolicyType policy;
+  return Load(filename, matrix, info, policy, fatal, transpose);
```

Comment:
Since you already provide an api to access the policy of the DatasetMapper, I think we could remove another Load function which allow user to pass the policy. This would make the api more consistent.

You can create the DatasetMapper in the Load function without the policy parameters pass

`info = info = DatasetMapper<PolicyType>(info.Policy(), cols);`

By this way users only need to store the state in their DatasetMapper only, I think this is less confusing.Else the users may think 
"I already store my policy in the DatasetMapper, why should I pass the policy into the Load function again?"

If you want to allow the users to get/set their policy states, you can add two api to the DatasetMapper

```
PolicyType const& Policy() const
{
  return policy;
}
void Policy(PolicyType value)
{
  policy = std::move(value);
}
```